### DISCUSSION TITLE
Running lazer on both os' on a dual booted drive

### DISCUSSION THREAD

**Discussion by The1Divider** [ppy/osu#17293] [Q&A] [answered]
Can I keep my `.exe` and `.appimage` in the same dir, will they use the same files/system? I'd rather not have a 2nd copy of everything on the same drive.

**Comment by peppy** [comment, marked as answer]
You can share the data directory by either symlinking or changing `storage.ini` on one of the two installations to point to the other one.

**Reply by The1Divider** [reply]
Where would that be located?

**Reply by peppy** [reply]
I'll leave that to you to figure out. If you can't find the file you likely won't be able to make this work either.
